app/views/layouts/application.html.erb

```diff
@@ -195,6 +195,22 @@
 </div>
 <script type="text/javascript" nonce="<%= content_security_policy_script_nonce %>">
 
+    // error handling for auth errors when getting user access tokens
+    if (window.SCP.userAccessToken === '' && <%= user_signed_in? %>) {
+        var errorTitle = 'Re-sign in again';
```

Comment:
'Re-sign in again' is seems redundant.  how about "Please refresh your login"?

Reply:
That was the suggestion from UX but I like your's better.

Reply:
I'd opt for "Please sign in again", for what it's worth.